### PR TITLE
create missing 'content' file

### PR DESCRIPTION
**Pull request by wfeldt**
If '/content' is missing on the ISO, create one.

### CODE REVIEW

File: mksusecd [hunk 1]
```diff
@@ -850,6 +852,50 @@ sub build_todo
...
+sub new_file
+{
+  my $fname = $_[0];
+  my $new_path = "$tmp_new/$fname";
+
+  if($fname =~ m#(.+)/([^/]+)#) {
+    system "mkdir -p '$tmp_new/$1'";
+  }
+
+  if(open my $x, ">$new_path") { close $x }
```
Comment (jreidinger) [inline mksusecd]: why not simply touch it? is this safer?

File: mksusecd [hunk 2]
```diff
@@ -2464,8 +2509,7 @@ sub sign_content
   my $c = copy_file "content";
   return if !defined $c;
 
-  my $k = copy_file "content.key";
-  return if !defined $k;
+  my $k = copy_or_new_file "content.key";
```
Comment (jreidinger) [inline mksusecd]: what about below `content.asc`? does it also make sense to create new one?

File: mksusecd (hunk 1 above)
Comment (wfeldt) [inline mksusecd]: I want to avoid running a shell just for that.

File: mksusecd (hunk 2 above)
Comment (wfeldt) [inline mksusecd]: Th signature file is re-created already.

**Comment by jreidinger** [comment]
LGTM, just is there any bnc or fate to reference it?

**Comment by wfeldt** [comment]
No. This came along with https://trello.com/c/wuyPuIyD.

**Pull request merged**